Cython/Utility/ModuleSetupCode.c

```diff
@@ -1767,6 +1767,20 @@ static CYTHON_INLINE int __Pyx_Is_Little_Endian(void)
   #define CYTHON_REFNANNY 0
 #endif
 
+#ifndef _ModuleSetupDecRef
+#if CYTHON_COMPILING_IN_LIMITED_API
+#define _ModuleSetupDecRef(obj) Py_DecRef(obj);
+#define _ModuleSetupXDecRef(obj) Py_DecRef(obj);
+#define _ModuleSetupIncRef(obj) Py_IncRef(obj);
+#define _ModuleSetupXIncRef(obj) Py_IncRef(obj);
+#else
+#define _ModuleSetupDecRef(obj) Py_DECREF(obj);
+#define _ModuleSetupXDecRef(obj) Py_XDECREF(obj);
+#define _ModuleSetupIncRef(obj) Py_INCREF(obj);
+#define _ModuleSetupXIncRef(obj) Py_XINCREF(obj);
+#endif
+#endif
```

Comment:
I've basically copy-pasted equivalent macros throughout this PR at the moment because I wasn't sure what order different snippets would get included into the file to know which macros would be defined when. Ideally these kinds of macros could be centralized somewhere. Is this module the best place to put something like that? Could this be patched into another `#else` block of the `#if CYTHON_REFNANNY` if-else below?

Reply:
> Ideally these kinds of macros could be centralized somewhere. Is this module the best place to put something like that?

Yes - ModuleSetupCode.c contains the bit of utility code that's always included, and is where the centralized stuff goes.

As I said in another comment - if we change it then I think think we should change it absolutely everywhere at once (with a big find/replace).

The naming scheme for Cython helper macros is `__Pyx_something` so they should stick to that. e.g. `__Pyx_Py_INCREF`

Reply:
I think `__Pyx_INCREF()` is enough.

Reply:
Going to resolve this since we've opened a separate issue to address the incref/decref changes.